tests/ci/codebuild/release/al2_aarch64_test.yml

```diff
@@ -0,0 +1,54 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 0.2
+
+env:
+  variables:
+    GO_TAG: go1.18.3
+phases:
+  # TODO: Everything in the install section should be moved into a prebuilt image
+  install:
+    runtime-versions:
+      java: corretto11
+    commands:
+      - yum -y update
+      - yum -y group install 'Development Tools'
+      - yum -y install cmake3
+      - yum -y install gsl-devel wget
+      - mkdir go_tmp
+      - cd go_tmp
+      - wget https://dl.google.com/go/$GO_TAG.linux-arm64.tar.gz
+      - tar -xvf $GO_TAG.linux-arm64.tar.gz
+      - mv go ../go_root
+      - cd ..
+  build:
+    commands:
+      # TODO: Move this to the docker image also
+      - export GOROOT=$(pwd)/go_root
+      - export GO111MODULE=on
+      - export PATH=$GOROOT/bin:$PATH
+      # TODO: Can we move the recursive pull logic upstream?
+      - git submodule update --init --recursive
```

Comment:
if we're pulling the JAR from sonatype staging (see `-DstagingProperties=true` below, L153 of build.gradle), why do we need to initialize the AWS-LC submodule? won't the compiled precompiled `.so` be bundled with the JAR?

isn't the goal of this buildspec to _not build anything_, instead pulling the JAR from staging and testing against that?

Reply:
got it. makes sense that we don't bundle test bytecode in our JAR and thus need to build it here.